packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -166,6 +168,8 @@ async function handleSuccessResponse(
   await blsCryptoClient.addSignature(partialSig, blindedQueryPhoneNumber, logger)
   // Send response immediately once we cross threshold
   if (!sentResult.sent && blsCryptoClient.hasSufficientVerifiedSignatures()) {
+    // Close outstanding requests
+    controller.abort()
```

Comment:
If we abort here, should we adjust our error reporting so that the "slowest" one doesn't look like they timed out?